### PR TITLE
fix a couple of bugs in evm/rpc module

### PR DESCRIPTION
**Pull request by louisliu2048**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by louisliu2048** [comment]
https://github.com/okex/ethermint/issues/483

**Comment by louisliu2048** [comment]
https://github.com/okex/ethermint/issues/482

File: app/rpc/namespaces/eth/api.go [hunk 1]
```diff
@@ -1032,11 +1032,14 @@ func (api *PublicEthereumAPI) generateFromArgs(args rpctypes.SendTxArgs) (*evmty
 	}
 
 	// Sets input to either Input or Data, if both are set and not equal error above returns
+	var data *hexutil.Bytes
 	var input []byte
```
Comment (xiangjianmeng) [inline app/rpc/namespaces/eth/api.go]: It is suggested to use `var input hexutil.Bytes` and remove `var data *hexutil.Bytes`

File: app/rpc/namespaces/eth/api.go [hunk 2]
```diff
@@ -1046,12 +1049,12 @@ func (api *PublicEthereumAPI) generateFromArgs(args rpctypes.SendTxArgs) (*evmty
...
 	if args.Gas == nil {
 		callArgs := rpctypes.CallArgs{
-			From:     &args.From,
+			From:     args.From,
 			To:       args.To,
 			Gas:      args.Gas,
 			GasPrice: args.GasPrice,
 			Value:    args.Value,
-			Data:     args.Data,
+			Data:     data,
```
Comment (xiangjianmeng) [inline app/rpc/namespaces/eth/api.go]: use `&input`

**Review by summerpro — DISMISSED** [review]

**Review by xiangjianmeng — APPROVED** [review]

**Pull request merged**